Blog post for MadAnalysis5 Multi-weight Integration (#1168)
HSF/hsf.github.io: _gsocblogs/2022/blog_MADAnalysis5-Multiweight-integration_KyleFan.md

---

project: MadAnalysis 5 Multiweight Integration
title: Midterm evaluation blog post
author: Kyle Fan
date: 24.07.2022 
year: 2022 
layout: blog_post
logo: MadAnalysis5-logo.png
intro: Midterm progress report

---

<p> 

> As of July 24th, 2022, with generous time and support from Jack and Benjamin, I have integrated multiweight container with the analysis platform. It currently fills in cutflows with multiweight data correctly. Changes are currently inside the multi_weight/multi_thread branch in my fork https://github.com/kfan326/madanalysis5/tree/multi_weight/multi_thread 

Reviewer: ```suggestion
As of July 24th, 2022, with generous time and support from Jack and Benjamin, I have integrated a multiweight container with the analysis platform. It currently fills in cut-flows with multiweight data correctly. Changes are currently held in [multi_weight/multi_thread](https://github.com/kfan326/madanalysis5/tree/multi_weight/multi_thread).
```
If you want to be more specific, you can include the full commit link since your branch will be updated; hence the progress information will be lost. PS `[...](...)` is markdown syntax for hyperlinks.
